README.md

```diff
@@ -15,7 +15,7 @@ $ apm install linter-golinter
 You must configure linter-golinter by editing ~/.atom/config.cson (choose Open Your Config in Atom menu):
```

Comment:
Can you modify this to mention the Settings menu? This text was probably written (or copied from one that was) before it existed.